### PR TITLE
Property-based Tests for valid_size and get_transaction_commitments

### LINKED ISSUE #12786: [zkapp_command]Write tests for valid_size and get_transaction_commitments
Write property-based tests for valid_size and get_transaction_commitments

### PR DESCRIPTION
**Pull request by ejMina226**
Property-based Tests for `valid_size` and `get_transaction_commitments` functions from `src/lib/mina_base/zkapp_command.ml`

* Closes #12786

### CODE REVIEW

**Comment by ejMina226** [comment]
!ci-build-me

**Comment by ejMina226** [comment]
!ci-build-me

**Comment by ejMina226** [comment]
!ci-build-me

**Comment by ejMina226** [comment]
!ci-build-me

**Review by mrmr1993 — APPROVED** [review]

File: src/lib/mina_base/test/zkapp_command/transaction_commitments.ml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      { Call_forest.Tree.account_update = account_gen
+      ; account_update_digest = ()
+      ; calls =
+          ([] : ( (Account_update.t, unit, unit) Call_forest.tree
+                , unit )
+                With_stack_hash.Stable.V1.t
+                list)
+      }
+
+    let zkapp_type_gen =
```
Comment (joaosreis) [inline src/lib/mina_base/test/zkapp_command/transaction_commitments.ml:21]: Maybe you could reuse here the generator defined in `src/lib/mina_base/test/zkapp_command/valid_size.ml` and just discard the `length`.

**Comment by ejMina226** [comment]
!ci-build-me

File: src/lib/mina_base/test/zkapp_command/transaction_commitments.ml (hunk 1 above)
Comment (ejMina226) [reply src/lib/mina_base/test/zkapp_command/transaction_commitments.ml]: I'll do this. Thanks.

**Comment by ejMina226** [comment]
!ci-build-me

**Comment by ejMina226** [comment]
!ci-build-me

**Comment by ejMina226** [comment]
!ci-build-me

**Pull request merged**